src/client/cli/cmd/unalias.cpp

```diff
@@ -32,8 +32,34 @@ mp::ReturnCode cmd::Unalias::run(mp::ArgParser* parser)
         return parser->returnCodeFrom(ret);
     }
 
-    if (aliases.remove_alias(alias_to_remove))
+    if (!parser->isSet(all_option_name))
     {
+        std::vector<std::string> bad_aliases;
+
+        for (const auto& alias_to_remove : aliases_to_remove)
+            if (!aliases.exists_alias(alias_to_remove))
+                bad_aliases.push_back(alias_to_remove);
+
+        if (!bad_aliases.empty())
+        {
+            auto i = bad_aliases.begin();
+            if (bad_aliases.size() == 1)
+                cerr << fmt::format("Unexisting alias: {}.\n", *i);
+            else
+            {
+                cerr << fmt::format("Unexisting aliases: {}", *i);
+                for (++i; i != bad_aliases.end(); ++i)
+                    cerr << fmt::format(", {}", *i);
+                cerr << ".\n";
+            }
```

Comment:
This feels a little awkward to me.  How about the following:
```c++
            cerr << fmt::format("Nonexistent {}: ", bad_aliases.size() == 1 ? "alias" : "aliases");

            for (const auto& bad_alias : bad_aliases)
            {
                cerr << fmt::format("{}{}", bad_alias, &bad_alias != &bad_aliases.back() ? ", " : "");
            }

            cerr << ".\n";
```

Reply:
How about:
```cpp
cerr << fmt::format("Nonexistent {}: {}", bad_aliases.size() == 1 ? "alias" : "aliases", fmt::join(bad_aliases, ", "));
```

Reply:
Oh @andrei-toterman, nice!